services/google-data-api/newrelic.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['basecms/google-data-api'],
```

Comment:
I know we haven't done this for our other services yet, but use the name field from package.json